app/src/main/java/org/wikipedia/settings/RecommendedReadingListNotificationManager.kt

```diff
@@ -0,0 +1,81 @@
+package org.wikipedia.settings
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import org.wikipedia.R
+import org.wikipedia.notifications.NotificationPollBroadcastReceiver
+import org.wikipedia.notifications.NotificationPollBroadcastReceiver.Companion.ACTION_RECOMMENDED_READING_LIST
+import org.wikipedia.notifications.NotificationPresenter
+import org.wikipedia.readinglist.recommended.RecommendedReadingListUpdateFrequency
+import java.time.Duration
+import java.time.LocalDateTime
+import java.time.LocalTime
+
+object RecommendedReadingListNotificationManager {
+    private const val NOTIFICATION_TYPE_LOCAL = "local"
+
+    fun scheduleRecommendedReadingListNotification(context: Context) {
+        val alarmManager = context.getSystemService(Context.ALARM_SERVICE) as AlarmManager
+        val intent = Intent(context, NotificationPollBroadcastReceiver::class.java)
+            .setAction(ACTION_RECOMMENDED_READING_LIST)
+        val triggerUpdateMillis = System.currentTimeMillis() + timeUntilNextUpdate(Prefs.recommendedReadingListUpdateFrequency).toMillis()
+        val nextUpdateIntervalInMillis = when (Prefs.recommendedReadingListUpdateFrequency) {
+            RecommendedReadingListUpdateFrequency.DAILY -> AlarmManager.INTERVAL_DAY
+            RecommendedReadingListUpdateFrequency.WEEKLY -> 7 * AlarmManager.INTERVAL_DAY
+            RecommendedReadingListUpdateFrequency.MONTHLY -> 30 * AlarmManager.INTERVAL_DAY
```

Comment:
Hardcoding 30 days per month feels inaccurate. Can we use similar logic to what's used in `RecommendedReadingListTask`?

Reply:
I think that's a good approach.